concepts/05 UI Components/Form/01 Getting Started with Form/40 Organize the Form Items/10 In Columns.md

```diff
@@ -0,0 +1,164 @@
+The Form can organize items into a fixed number of columns or automatically adjust the layout based on the screen width. To set a fixed number of columns, initialize the [colCount](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#colCount) property as done in the code below. To create an adaptive layout, configure the [screenByWidth](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#screenByWidth) and [colCountByScreen](/Documentation/ApiReference/UI_Components/dxForm/Configuration/colCountByScreen/) properties. 
+
+An item can span multiple columns. The example below sets [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) property for the `Notes` item so that it spans two columns.
```

Comment:
```suggestion
An item can span multiple columns. The example below sets the [colSpan](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/SimpleItem/#colSpan) property for the `Notes` item so that it spans two columns.
```

Reply:
"sets the [colSpan] property" Sets it to what?

Reply:
will fix it, 

Sets the colSpan property for the `Notes` items to `2` so that it spans two columns.